.github/workflows/ci.yml

```diff
@@ -24,9 +24,18 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+      - name: Clone CCM
+        uses: actions/checkout@v2
+        with:
+          repository: dimas-b/ccm
+          ref: increase-start-timeout
+          fetch-depth: 1
+          path: ccm
+      - name: Install CCM Dependencies
+        run: pip install --user setuptools pyYaml six tailer ldap3 lxml
       - name: run tests and analyze
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-        run: mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar --file pom.xml
+        run: env PATH=$PWD/ccm:$PATH mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar --file pom.xml
```

Comment:
Shouldn't we use `sudo ./setup.py install` ?

Reply:
Good point. I'll give it a try... I basically copied python commands from some Jenkins job I had before, but this approach looks cleaner.

Reply:
Updated and force pushed - seems to work fine. thanks for the tip :)